vendor.conf

```diff
@@ -1,7 +1,8 @@
 github.com/BurntSushi/toml master
 github.com/Nvveen/Gotty master
 github.com/blang/semver master
-github.com/containers/image 063852766c3e82ec8359ce5f6612e056f3efaa76
+# github.com/containers/image 063852766c3e82ec8359ce5f6612e056f3efaa76
+github.com/containers/image dir_transport https://github.com/umohnani8/image
```

Comment:
Will this change before you do a final submit?

Reply:
yes, once it gets accepted to upstream I will change the vendor.